docs/videos/dev.md

```diff
@@ -0,0 +1,13 @@
+# Overview
+
+The All Core Devs meeting is a public meeting of Aragon core developers. All developers working on all layers of the Aragon stack are welcome to join if they have an agenda item they'd like to discuss. The meetings are recorded, published, and documented here on the wiki. You can find documentation for past meetings under the All Core Devs section of the Aragon Wiki.
+
+# Participating in the All Core Devs meeting
+To participate in the call, you only have to have an agenda item accepted by one of the call organizers. These are the steps for proposing an agenda item:
+
+- Submit a pull request that adds a bullet point to the agenda linked in the Next Meeting section below.
```

Comment:
Just worried this might be too much at this point, compared to commenting on a google doc that can be added here in the wiki.

Reply:
My thought is, the less moving parts the better. Totally open to making it a Google Doc comment vs Pull Request if you think that will ease / increase participation.